src/KubernetesClient/TimeoutHandler.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.WebUtilities;
+
+namespace informers
+{
+    public class TimeoutHandler : DelegatingHandler
```

Comment:
Can you document what this is used for?

Reply:
Added description to the class that explains its purpose and how to use